elements/lisk-p2p/src/utils/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+export * from './miscellaneous';
```

Comment:
if you look at this file, it is all related to IP address related functionality and not really miscellaneous. perhaps we can name it `net` or something of the sort

Reply:
There is also `getNetwork` , `getBucket`, `getByteSize`... I thought `miscellaneous` can englobe any case.

Reply:
all of those you mentioned are related to or only used in the other ip address related functions.

Reply:
We don't `network` anything in those methods. What we do is more `get`, `compute` or `calculate`. Therefore, I prefer to keep `misc` since I don't find a proper verb to follow the pattern.

Reply:
Can we use `helpers` maybe?

Reply:
It looks a bit redundant `utils/helpers.ts`.